app/views/dashboards/show.html.erb

```diff
@@ -1 +1,5 @@
-
+<div>
```

Comment:
well, my current thought process is to put everything into a div, maybe so I could center this content on the page. I was thinking about including some simple css/sass as I go, mostly just layout stuff, centering this content or something, but I have not decided one way or the other. thoughts?